ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -258,13 +258,14 @@ Launch And Access Workbench From Projects Home Page
     [Documentation]    Launches a running workbench from DS Projects home page.
     [Arguments]     ${workbench_title}    ${project_title}    ${username}=${TEST_USER_3.USERNAME}
     ...             ${password}=${TEST_USER_3.PASSWORD}    ${auth_type}=${TEST_USER_3.AUTH_TYPE}
+    ...             ${expected_ide}=JupyterHub
```

Comment:
pls change it to Jupyter or JupyterLab

Reply:
Of course, updated! :slightly_smiling_face: